index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    <title>Who's Anthony? Who's Anthony?</title>
+  </head>
+  <body>
+    <h1>Anthony Robinson</h1>
+
+    <h2>About</h2>
+        <section>
```

Comment:
you have an extra two levels of indentation from line 13-21.  The `<h2>` and the `<section>` below it are siblings, so they should be indented to the same level.  also, you use 4 spaces instead of 2 spaces